EIP-draft.md

```diff
@@ -262,7 +261,7 @@ The reference vesting NFT repository includes tests written in Hardhat.
 
 ## Reference Implementation
 
-A reference implementation of this EIP can be found in [eip-5725 assets](../assets/eip-5725/README.md/).
+A reference implementation of this EIP can be found in [eip-5725 assets](../assets/eip-5725/README).
```

Comment:
Please leave `.md`. (ex: `../assets/eip-5725/README.md`), as the review was just mentioning the end `/`